linera-service/src/server.rs

```diff
@@ -317,6 +326,12 @@ enum ServerCommand {
         /// The maximal number of entries in the storage cache.
         #[structopt(long, default_value = "1000")]
         cache_size: usize,
+
+        /// Use this when running from a Kubernetes cluster (including from within GCP)
+        /// Won't use config files to determine host IPs, but will do it dynamically
```

Comment:
Shouldn't we use different configuration files without host IPs in the first place then?

Reply:
So this is actually not very well phrased. We do still use the config files, but the hosts won't have IPs, they'll have Kubernetes pod names instead. Which we'll use to fetch the actual IPs. I can update this comment